source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRPopoverTarget.h

```diff
@@ -0,0 +1,39 @@
+//
+//  ACRPopoverTarget.h
+//  AdaptiveCards
+//
+//  Created by Jitisha Azad on 01/07/25.
+//  Copyright © 2025 Microsoft. All rights reserved.
+//
+
+#import "ACOInputResults.h"
+#import "ACRBaseTarget.h"
+#import <AdaptiveCards/ACOInputResults.h>
+#import <AdaptiveCards/ACRBaseTarget.h>
+
+#import "ACRIContentHoldingView.h"
+#import "ACRView.h"
+#import <UIKit/UIKit.h>
+
+@interface ACRPopoverTarget : ACRBaseTarget
+
+@property ACOBaseActionElement *actionElement;
+@property (weak) ACRView *rootView;
```

Comment:
If there is no good reason for these properties to be atomic please mark them as `nonatomic`.